CONTRIBUTING.md

```diff
@@ -31,10 +32,35 @@ A maintainer will request changes if necessary. For content contributions, don't
 
 ## Setup Instructions
 
-This project is built using [Hugo](https://gohugo.io/), an open-source static site generator. To get the project up and running:
+This project is built using
+
+- [Hugo](https://gohugo.io/), an open-source static site generator
+- [Docker](https://www.docker.com/), a platform to generate containerized environments.
+
+Using Docker for the project is preferred to minimize conflicts arising from machine, OS, or dependency variations.
```

Comment:
Let's say "using Docker for developing this project can help minimize conflicts [rest of sentence]"